README.md

```diff
@@ -74,7 +74,40 @@ The `useFloating` Svelte hook acts as a controller for all other Floating UI Sve
 
 ### useInteractions
 
-(tbd)
+The `useInteractions` Svelte hook allows you to consume multiple interactions. It ensures that event listeners from different hooks are properly registered instead of being overruled by one another.
+
+#### Usage
+
+```html
+<script>
+	import { useFloating, useHover, useFocus, useInteractions } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
+
+	const hover = useHover(floating.context);
+	const focus = useFocus(floating.context);
+
+	const interactions = useInteractions([hover, focus]);
+</script>
+
+<div {...interactions.getReferenceProps()}>Reference</div>
+<div {...interactions.getFloatingProps()}>Floating</div>
+```
+
+If you want to apply an event handler the an element using a props getter make sure to pass them through the getter instead of applying them directly:
+```diff
+- <div {...interactions.getReferenceProps()} onclick={() => console.log('click!')}>Reference</div>
++ <div {...interactions.getReferenceProps({ onclick: () => console.log('click!') })}>Reference</div>
```

Comment:
Be mindful how line length for the README, it does have a fixed width:

![Screenshot 2024-05-03 at 12 14 25 PM](https://github.com/skeletonlabs/floating-ui-svelte/assets/1509726/bc41a096-384c-4e94-bac6-345178c0c136)

I'd suggest one of two changes:

1. Abstract the click handler to a function, which can shorten the length
2. Wrap to multiple lines

Number one seems preferable!